slick/src/main/scala/slick/jdbc/SQLiteProfile.scala

```diff
@@ -98,14 +98,24 @@ trait SQLiteProfile extends JdbcProfile {
     override def createColumnBuilder(tableBuilder: TableBuilder, meta: MColumn): ColumnBuilder = new ColumnBuilder(tableBuilder, meta) {
       /** Regex matcher to extract name and length out of a db type name with length ascription */
       final val TypePattern = "^([A-Z]+)(\\(([0-9]+)\\))?$".r
+      final val TimestampPattern = """^(?:"(.*)")|([0-9]+)""".r
       private val (_dbType,_size) = meta.typeName match {
         case TypePattern(d,_,s) => (d, Option(s).map(_.toInt))
       }
       override def dbType = Some(_dbType)
       override def length = _size
       override def varying = dbType == Some("VARCHAR")
-      override def default = meta.columnDef.map((_,tpe)).collect{
+      override def default = meta.columnDef.map((_,tpe)).collect{ 
         case ("null",_)  => Some(None) // 3.7.15-M1
+        case (v , "java.sql.Timestamp") => Some(Some( v match{
+            case TimestampPattern(ts,ts_unix) => if( ts != null  ){
+                if(ts contains ":" )
+                  if( ts contains "-") java.sql.Timestamp.valueOf(ts) else new java.sql.Timestamp(  java.sql.Time.valueOf(ts).getTime )
+                else if( ts contains "-" )
+                  new java.sql.Timestamp(java.sql.Date.valueOf(ts).getTime)    
```

Comment:
How about the timestamp of a specific time zone? like `2013-10-07T08:23:19.120Z` and `2013-10-07 04:23:19.120-04:00` shown [here](https://www.sqlite.org/lang_datefunc.html).